tools/turbinia.service

```diff
@@ -0,0 +1,29 @@
+# This file should be in /etc/systemd/system/turbinia.service
+# sudo systemctl enable turbinia
+# sudo systemctl status turbinia
+# sudo systemctl start turbinia
+# sudo systemctl stop turbinia
+# sudo systemctl daemon-reload
+
+[Unit]
+Description=Turbinia Daemon
+Documentation=https://github.com/google/turbinia
+Before=multi-user.target
+After=network.target
+After=remote-fs.target
+
+[Service]
+Type=simple
+PIDFile=/var/run/turbinia-daemon.pid
+User=turbinia
+Group=turbinia
+Restart=always
+RestartSec=10
+ExecStartPre=/bin/sh -c 'sudo systemctl set-environment GOOGLE_APPLICATION_CREDENTIALS=/home/turbinia/turbinia-service-account-creds.json'
```

Comment:
Is this the right way to set environment variables in this case?  I don't know systemd much, but a quick search shows that there's also an 'Environment' key that can be set.

Reply:
The problem with the Environment key is that it needs to be hardcoded, which is why i'm using execstartpre.